crates/sncast/src/main.rs

```diff
@@ -149,6 +151,9 @@ fn main() -> Result<()> {
         print_command_result("script", &mut result, numbers_format, &output_format)?;
         Ok(())
     } else {
+        let mut config = load_config(&cli.profile, &None)?;
```

Comment:
improving this will be a part of https://github.com/foundry-rs/starknet-foundry/issues/1329